src/childinfo.c

```diff
@@ -112,17 +115,21 @@ void sendChildInfoGeneric(childInfoType info_type, size_t keys, double progress,
 
 /* Update Child info. */
 void updateChildInfo(childInfoType information_type, size_t cow, monotime cow_updated, size_t keys, double progress) {
+    if (cow > server.stat_current_cow_peak) {
+        server.stat_current_cow_peak = cow;
+    }
```

Comment:
```suggestion
    if (cow > server.stat_current_cow_peak)
        server.stat_current_cow_peak = cow;
```
I still remember that it was you who reminded me.

Reply:
I remembered i said i prefer to keep them in one line if the length is short

Reply:
T_T, I may have misremembered.

Reply:
Actually, in redis, this style is not clear, i just make typesetting tidy. if it is not long, i prefer one line, otherwise, i prefer to use {}, of course, sometimes, i follow old style around modified code block.

Reply:
I've gotten used to single lines without `{`.

Reply:
Some styles should have specification. I always curious about how to use. 
* `if` one line or two lines, with or without `{ }`;
* spaces between operators or not;
* spaces between function call params or not.

Reply:
@huangzhw Like @ShooterIT said, I also follow old style around modified code block.

Reply:
This is only way we can follow.

Reply:
my preference is for less lines, especially around trivial things or error handling, don't like to hide the important things by wasting a lot of lines on nonsense.
as you said, there are many styles in redis, so IMHO this specific topic isn't strict.